model/user.go

```diff
@@ -244,8 +244,8 @@ func (u *User) ToJson() string {
 }
 
 // Generate a valid strong etag so the browser can cache the results
-func (u *User) Etag() string {
-	return Etag(u.Id, u.UpdateAt)
```

Comment:
Yeah, I think that's fine for the time being since most of the objects have Etags based off of just their Id and UpdateAt fields, but I kind of expected that a pre-sanitized user would have a different etag than a sanitized one because they contain different data, similar to a hash of the user object. We can always deal with that if it ever becomes an issue though